app/src/test/java/org/mozilla/tv/firefox/helpers/RxSchedulerBeforeClass.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import io.reactivex.Scheduler
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.functions.Function
+import io.reactivex.plugins.RxJavaPlugins
+import io.reactivex.schedulers.Schedulers
+import io.reactivex.schedulers.TestScheduler
+import org.junit.BeforeClass
+import org.junit.rules.ExternalResource
+import java.util.concurrent.Callable
+
+/**
+ * Used to force all tested Rx code to execute synchronously.
+ *
+ * IMPORTANT NOTE: **must** be called from [BeforeClass]
+ */
+fun forceRxSynchronous() {
```

Comment:
nit: why are these declared at the top-level? In general, I'd prefer to namespace things inside a class or object.

Reply:
My bad, I always forget that.